### PR TITLE
Mapping `workflow_dispatch` inputs into the Expression `inputs` context

### PR DESCRIPTION
**Pull request by KnisterPeter**
This changes adds the workflow_dispatch event inputs to the `inputs` context and maintaining the boolean type.

### CODE REVIEW

**Comment by KnisterPeter** [comment]
You are right regarding the input types. We need to coerce them.

**Comment by KnisterPeter** [comment]
This should reflect githubs behavior now as well

File: pkg/runner/testdata/workflow_dispatch/workflow_dispatch.yml [hunk 1]
```diff
@@ -27,6 +27,10 @@ jobs:
         run: |
           echo input.with_default=${{ inputs.with_default }}
           [[ "${{ inputs.with_default }}" = "default" ]] || exit 1
-      - name: test boolean input
-        if: ${{ !inputs.boolean }}
-        run: exit 1
+      - id: boolean-test
+        name: run on boolean input
+        if: ${{ inputs.boolean == true }}
```
[the remark this answers is not in the document]
Comment (KnisterPeter) [reply pkg/runner/testdata/workflow_dispatch/workflow_dispatch.yml:32]: But `inputs.boolean` should be a real boolean not a string.
That should be converted by these lines:

https://github.com/nektos/act/blob/15ee53088c724bcf32eab72e788d22cdf2c50bdf/pkg/runner/expression.go#L330-L334

Am I missing something? :thinking:

**Review by cplee — DISMISSED** [review]

**Comment by KnisterPeter** [comment]
Oh I see. We are depending on code which is not managed by the act team and that did change.
Not a good state I would say.

**Comment by KnisterPeter** [comment]
I did re-run while the CI indicates that two tests failed (https://github.com/nektos/act/actions/runs/3263285979/jobs/5362282276#step:6:33010) and both are fine locally. But anyway if the error lies in the external code I guess I will wait with this PR until its fixed.

**Review by cplee — APPROVED** [review]

**Comment by itsjoekent** [comment]
Any chance this can get released soon? Looks like the last release was 19 days ago

**Comment by KnisterPeter** [comment]
There is one usual release per month

**Pull request merged**